src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -93,13 +93,18 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           <div id="tooltip">
             <div class="title">${computeStateName(stateObj)}</div>
             <div class="current-temperature">
```

Comment:
This whole div doesnt need to be rendered if there is not current temp

Reply:
As said above the layout get's messed up then.

Reply:
Then we should add a class that add margin/padding